### ISSUE TITLE
 Build failed with an exception on 3.4.0

### ISSUE THREAD

**Issue by arnabonetraker** [juliansteenbakker/mobile_scanner#715]
Flutter Build Error with `package 3.4.0`
```
/mobile_scanner-3.4.0/lib/src/mobile_scanner.dart:251:34: Error: Member not found: 'MediaQuery.sizeOf'.
    final Size size = MediaQuery.sizeOf(context);
                                 ^^^^^^
Target kernel_snapshot failed: Exception


FAILURE: Build failed with an exception.
```

**Comment by mustafayilmazdev** [comment]
Same here!

**Comment by juliansteenbakker** [comment]
Fixed in v3.4.1
